x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_charts_container.js

```diff
@@ -191,6 +265,23 @@ function ExplorerChartContainer({
                 </EuiButtonEmpty>
               </EuiToolTip>
             )}
+            {chartType === CHART_TYPE.GEO_MAP && mapsLink ? (
+              <EuiToolTip position="top" content={openInMapsPluginMessage}>
```

Comment:
Not needed for this PR, but would be great to add a `View in Maps` item to the Actions menu in the anomalies table, similar to the `View series` link for time series anomalies which is available for the anomaly charts and the anomalies table.